react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: []
+    };
```

Comment:
Try reworking your solution to store the `openIndex` instead of which topics are clicked.

```js
this.state = {
  openIndex: null
};
```

Then, when the user clicks a topic, update the `openIndex`. When you render the topics, you can use the `index` parameter of your `.map()` callback to conditionally show/hide each topic as its react elements are created.

If the user clicks the topic that is already open, set `openTopic` back to `null`.